### PR TITLE
Quantize nearest_interp and nearest_interp_v2

### PR DESCRIPTION
**Pull request by wozna**
### PR types

Performance optimization
### PR changes

OPs 
### Describe

This PR adds: 
- quantization for the nearest_interp and the nearest_interp_v2 for QAT and PTQ method,
- unit tests,
- in `cpu_quantize_placement_pass.cc`  added checking if the list of operators entered by the user contains all the operators supported by quantization

Nearest_interp quantization improved the performance by 2% in the "faster_rcnn" model and almost 11% in the "ocr_det" model.

### CODE REVIEW

File: paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc [hunk 1]
```diff
@@ -23,15 +24,34 @@ class Graph;
 
 void CPUQuantizePlacementPass::ApplyImpl(ir::Graph* graph) const {
   VLOG(3) << "Marks operators which are to be quantized.";
+  std::unordered_set<std::string> supported_op_types =
+      std::unordered_set<std::string>(
+          {"concat", "conv2d", "depthwise_conv2d", "elementwise_add", "fc",
+           "matmul", "nearest_interp", "nearest_interp_v2", "pool2d",
+           "prior_box", "reshape2", "transpose2", "fusion_gru", "fusion_lstm",
```
Comment (jakpiase) [inline paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc:31]: Just for my curiosity, what is `prior_box`? Do we support that operation?

Comment (jakpiase) [reply paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc:31]: I have found it already, I think that it is the only op that doesn't end with "mkldnn_op.cc"

**Review by jakpiase — APPROVED** [review]
LGTM

File: paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc (hunk 1 above)
Comment (wozna) [reply paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc:31]: Yes, you are right, we are using a native version here which supports all data types including int8.

**Review by lidanqing-vv — APPROVED** [review]
LGTM.
This PR supported Nearest_interp int8 and improved "faster_rcnn" int8 model by 2%, ocr_det int8 model by 11%, Retinanet int8 model by 2%

**Comment by lidanqing-vv** [comment]
@baoachun Can I merge this PR?

**Review by baoachun — APPROVED** [review]
LGTM

**Review by lidanqing-vv — APPROVED** [review]
LGTM

**Comment by lidanqing-vv** [comment]
@Aganlengzi Hi Baoachun approved. Please merge this PR, Thanks !

**Pull request merged**